amethyst_input/src/input_handler.rs

```diff
@@ -213,6 +213,18 @@ where
                 } => {
                     event_handler.single_write(MouseMoved { delta_x, delta_y });
                 }
+                DeviceEvent::MouseWheel {
```

Comment:
Won't there be both events emitted?

Reply:
No, there wont't be both of them emitted at the same time, as mentioned here:

> 
> 
> We actually shall not care about both events happening at the same time, because their invoking [implementation](https://github.com/tomaka/winit/blob/089816d9bab498c0ec002d7e560ad940b78d476e/src/platform/linux/wayland/pointer.rs#L146) will trigger only one of them at any time.

Reply:
Oh, sorry for asking the same question twice.